packages/flutter_driver/lib/src/common/wait.dart

```diff
@@ -0,0 +1,325 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:convert';
+
+import 'package:flutter/scheduler.dart';
+import 'package:flutter/widgets.dart';
+
+import 'message.dart';
+
+/// A Flutter Driver command that waits until a given [condition] is satisfied.
+class WaitForCondition extends Command {
+  /// Creates a command that waits for the given [condition] is met.
+  const WaitForCondition(this.condition, {Duration timeout})
+      : assert(condition != null),
+        super(timeout: timeout);
+
+  /// Deserializes this command from the value generated by [serialize].
+  ///
+  /// The [json] argument cannot be null.
```

Comment:
I believe elsewhere in flutter we use the phrase "must not be null". We should stick with that.

Reply:
I've seen both... but I updated all the usage here to "must not be null".